tools/distrib/clang_tidy_code.sh

```diff
@@ -21,6 +21,16 @@ set -ex
 cd $(dirname $0)/../..
 REPO_ROOT=$(pwd)
 
+# generate a clang compilation database
+tools/distrib/gen_compilation_database.py \
+  --include_headers \
+  --ignore_system_headers \
+  "//:*" \
+  "//src/core/..." \
```

Comment:
Should this also cover `//src/cpp/...`?

Reply:
I like to do it although there are no targets under //src/cpp/... but blaze throws an error if the expression doesn't have any.